pkgs/by-name/ro/rollo-x1038/package.nix

```
@@ -0,0 +1,30 @@
+{
+  stdenv,
+  cups,
+  lib,
+}:
+
+stdenv.mkDerivation {
+  name = "rollo-x1038";
+  version = "1.8.4";
+  src = fetchTarball {
+    url = "https://rollo-main.b-cdn.net/driver-dl/linux/rollo-cups-driver-1.8.4.tar.gz";
+    sha256 = "sha256:0x8dd6wj5n974k3vb3iir61dd46ag13zj7s9dhgv722cky7v57h4";
```

```suggestion
    hash = "";
```
1. `hash` is preferred over type-specific fields
2. you'll need to regenerate hash (hint: you `nix-build` the package, see the hash mismatch error and copy the second one into the file)